Orange/base.py

```diff
@@ -18,13 +19,12 @@ class Learner:
     name = 'learner'
     #: A sequence of data preprocessors to apply on data prior to
     #: fitting the model
-    preprocessors = ()
+    preprocessors = None
     learner_adequacy_err_msg = ''
 
     def __init__(self, preprocessors=None):
-        if preprocessors is None:
-            preprocessors = type(self).preprocessors
-        self.preprocessors = list(preprocessors)
+        self.use_default_preprocessors = False
```

Comment:
Right, thanks. I think this explanation along with the one given in https://github.com/biolab/orange3/pull/1657#issuecomment-263121263 should be added as a comment there somewhere. Otherwise I'll need to rediscover this repeatedly. :smiley: